app/assets/stylesheets/charts.scss

```diff
@@ -1,157 +0,0 @@
-.stats-well {
-  border: 1px solid #d7d7d7;
-  background-size: 100%;
-  padding: 15px;
-  min-width: 280px;
-  min-height: 100px;
-  margin-bottom: 15px;
-
-  h2,
-  h4 {
-    font-weight: normal;
-  }
-}
```

Comment:
This class is still used in some places

Reply:
I hope I've done all that are still needed.